zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/JSONParser.scala

```diff
@@ -1,34 +1,65 @@
 package edu.uci.ics.cloudberry.zion.model.impl
 
-import edu.uci.ics.cloudberry.zion.model.datastore.{IJSONParser, JsonRequestException}
+import edu.uci.ics.cloudberry.zion.model.datastore.{FieldNotFound, IJSONParser, JsonRequestException, QueryParsingException}
```

Comment:
Maybe we can separate QueryValidator from it? This one can be treated separately, and more validation login could be added later.